From: Sam Hartman <hartmans@debian.org>
Date: Tue, 14 Jan 2025 15:04:41 -0700
Subject: Libpam is both shared and static

---
 libpam/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libpam/meson.build b/libpam/meson.build
index 3e8a531..95a6054 100644
--- a/libpam/meson.build
+++ b/libpam/meson.build
@@ -43,7 +43,7 @@ libpam_map_path = meson.current_source_dir() / libpam_map
 libpam_link_deps = [libpam_map]
 libpam_link_args = ['-Wl,--version-script=' + libpam_map_path]
 
-libpam = shared_library(
+libpam = both_libraries(
   'pam',
   sources: libpam_src,
   include_directories: [libpam_inc],
